dnase/trackhub/update_tracks.bash

```diff
@@ -213,6 +213,7 @@ cp -R assets/${hub_type}/. ${hub_target}
 
 for i in "${genome_array[@]}"; do
```

Comment:
is i an integer?

Reply:
No.  i is hg38, mm10, rn6, etc.

Reply:
then rename to something more descriptive like cur_genome

use i only for integers